### DISCUSSION TITLE
Choose the highest resolution from assets?

### DISCUSSION THREAD

**Discussion by four43** [developmentseed/cogeo-mosaic#217] [Ideas]
I have a global data set that is comprised of many files that I've pre-computed global z0-z8 COG for. I'd love to use that until I get to Zoom 9, then fall back to the rest of the mosaic. Is that something that I should just implement as two different mosaics (just to keep the interface the same, one for z0-8 and one for 9+) or is there a sorting method available for that?

Thanks for the great work on all of this, this is really cool!

**Comment by vincentsarago** [comment]
👋 @four43 

> Is that something that I should just implement as two different mosaics

At one point we wanted to address this in mosaicJSON https://github.com/developmentseed/mosaicjson-spec/issues/6 but didn't pursue. The best way is definitely two have 2 mosaics and then in your application switch between both depending on the zoom level

**Comment by four43** [comment]
Hey that sounds great. I appreciate the response. We'll continue in that direction.
